File: src/main/java/me/freitaseric/commands/utils/PingCommand.java
package me.freitaseric.commands.utils;

import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;

public class PingCommand {
    public void run(SlashCommandInteractionEvent event) {
        long ping = event.getJDA().getGatewayPing();
        event.reply(String.format("Gateway ping is `%sms`!", ping)).setEphemeral(true).queue();
    }
}

File: src/main/java/me/freitaseric/commands/utils/PingCommand.kt
package me.freitaseric.commands.utils

import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent

class PingCommand(private val event: SlashCommandInteractionEvent) {
    init {
        val ping: Long = event.jda.gatewayPing
        event.reply(String.format("Gateway ping is `%sms`!", ping)).queue()
    }
}